docs/maui/essentials/badge.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Badge - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: The Badge allows developers to set the app icon badge number on the homescreen.
+ms.date: 07/14/2023
+---
+
+# Badge
+
+The `Badge` API allows developers to set the app icon badge number on the homescreen.
+
+![Screenshot of an Badge on Windows](../images/essentials/badge-windows.png "Badge on Windows")
+
+---
+
+The following preconditions required for the `Badge` API:
+
+# [iOS/MacCatalyst](#tab/ios)
```

Comment:
Let's add an Android tab that says "Additional configuration required to support Android devices" and links to the [`Android` section below](https://github.com/MicrosoftDocs/CommunityToolkit/pull/298/files#diff-3731d113b40a7c5539b3c9113f1f47e06e016f545f5ea94f5c830a2e483deba1R100-R137).

Otherwise, devs may see that there's no "preconditions required" for Android and never scroll down to the [`Android` section below](https://github.com/MicrosoftDocs/CommunityToolkit/pull/298/files#diff-3731d113b40a7c5539b3c9113f1f47e06e016f545f5ea94f5c830a2e483deba1R100-R137).